web/src/components/users/RootAuthMethods.jsx

```diff
@@ -20,14 +20,26 @@
  */
 
 import React, { useState, useEffect } from "react";
-import { Skeleton, Truncate } from "@patternfly/react-core";
+import { Button, Skeleton, Truncate } from "@patternfly/react-core";
 import { TableComposable, Thead, Tr, Th, Tbody, Td } from '@patternfly/react-table';
 import { Em, RowActions } from '~/components/core';
 import { RootPasswordPopup, RootSSHKeyPopup } from '~/components/users';
 
 import { useCancellablePromise } from "~/utils";
 import { useInstallerClient } from "~/context/installer";
 
+const MethodsNotDefined = ({ setPassword, setSSHKey }) => {
+  return (
+    <div className="stack">
+      <div className="bold">No root auth method defined yet</div>
+      <div>Please, define at least one root authentication method for being able to log into the system as administrative user.</div>
+      <div className="split">
+        <Button variant="primary" onClick={setPassword}>Set a password</Button>
+        <Button variant="secondary" onClick={setSSHKey}>Upload a SSH Public Key</Button>
```

Comment:
No strong opinion at this point from my side here. Just setting one action as primary and the other as secondary for not crowding the UI of primary actions.

| Both as primary action | Only "Set a password" as primary action |
|-|-|
| ![Screen Shot 2023-06-12 at 12 10 49](https://github.com/openSUSE/agama/assets/1691872/a91cf20e-1bef-414b-8065-3d0abac452f5) | ![Screen Shot 2023-06-12 at 12 04 19](https://github.com/openSUSE/agama/assets/1691872/26ca3842-48fb-4500-95a4-e7cf78304263) |

```suggestion
        <Button variant="primary" onClick={setPassword}>Set a password</Button>
        <Button variant="primary" onClick={setSSHKey}>Upload a SSH Public Key</Button>
```

Reply:
I visually prefer the version with primary and secondary buttons. And I would say it makes sense to offer the password as primary (it is the most common).